src/main/java/amidst/gui/profileselect/LocalProfileComponent.java

```diff
@@ -81,6 +82,13 @@ private boolean tryLoad() {
 		try {
 			Log.i("using minecraft launcher profile '" + getProfileName() + "' with versionId '" + getVersionName()
 					+ "'");
+					
+			String possibleModProfiles = "optifine|forge";
+            if (Pattern.matches(possibleModProfiles, getProfileName().toLowerCase())) {
```

Comment:
`Pattern#matches` requires the complete profile name to match the pattern. Thus, this only works if the complete profile name is either the word "optifine" or "forge", without anything else. For example "OptiFine 1.8.9" does not work. My suggestion is to simply use a list of strings and check if the profile name `contains` any of the strings.

`String#toLowerCase` depends on the default `Locale` on the users computer. This is an issue, see [here](https://github.com/toolbox4minecraft/amidst/issues/151#issuecomment-211903792) and [here](http://stackoverflow.com/questions/11063102/using-locales-with-javas-tolowercase-and-touppercase). However, it should work to use `.toLowerCase(Locale.ENGLISH)`.